tests/test_utils_reqser.py

```diff
@@ -53,6 +54,7 @@ def _assert_same_request(self, r1, r2):
         self.assertEqual(r1._encoding, r2._encoding)
         self.assertEqual(r1.priority, r2.priority)
         self.assertEqual(r1.dont_filter, r2.dont_filter)
+        self.assertEquals(r1.flags, r2.flags)
```

Comment:
According to https://docs.python.org/3.5/library/unittest.html#deprecated-aliases assertEquals is a deprecated alias to assertEqual - could you please use assertEqual, like in above lines?